src/components/forms/profile/update-user-profile-form.tsx

```diff
@@ -80,9 +94,14 @@ const UpdateUserProfileForm: React.FC<{ userProfileData?: Profile }> = ({
             userProfile.birthdate !== "" ? userProfile.birthdate : undefined
           }
           onChange={(date) => {
-            setUserProfile({
-              ...userProfile,
-              birthdate: date.toLocaleDateString("en-EN"),
+            setUserProfile((prevState) => {
+              return {
+                ...userProfile,
+                birthdate:
+                  date !== null
+                    ? date.toLocaleDateString("en-EN")
+                    : prevState.birthdate,
+              };
```

Comment:
I think it was mainly used before with React class components, and not so much used since React hooks are democratized, but in this case, it is really helpful 🙂